src/sql/base/browser/ui/dropdownList/dropdownList.ts

```diff
@@ -79,7 +79,7 @@ export class Dropdown extends BaseDropdown {
 	}
 }
 
-export class DropdownList extends BaseDropdown {
+export class DropdownList extends Dropdown {
```

Comment:
Have you tested this with other places that use the control (if any) to make sure the behavior there is still working correctly?

Is this something that broke in the merge?

Reply:
Looks like it was - @kburtram do you recall why this was changed to extend BaseDropdown? Let's make sure we're not regressing something else that necessitated that change in the first place.